ES/ESmy020.xml

```diff
@@ -108,97 +431,229 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                              referring to the content of the manuscript.</desc>
+                              <q xml:lang="gez"><hi rend="rubric">ቡራኬ፡ ዘአቡነ፡ ሳሙኤል፡</hi></q>
+                        </item>
+
+                        <item xml:id="e12">
+                          <locus target="#49r"/>
+                           <desc type="findingAid">Note in the upper margin, in <ref target="#h1">Hand 1</hand>,
+                              referring to the content of the manuscript.</desc>
+                              <q xml:lang="gez">ቡራኬ፡ ዘአቡነ፡ ሳሙኤል፡</q>
+                        </item>
+
+                        <item xml:id="e13">
+                          <locus target="#60r"/>
+                           <desc type="findingAid">Note in the upper margin, in <ref target="#h1">Hand 1</hand>,
+                              referring to the content of the manuscript.</desc>
+                              <q xml:lang="gez">ሐዋርያ፡</q>
+                        </item>
+
+                        <item xml:id="e14">
+                          <locus target="#69r"/>
+                           <desc type="findingAid">Note in the upper margin, in <ref target="#h1">Hand 1</hand>,
+                              referring to the content of the manuscript.</desc>
+                              <q xml:lang="gez">ሰርሖማይ፡</q>
+                        </item>
+
                      </list>
                   </additions>
                   <bindingDesc>
-                     <binding contemporary="partly">
-                        <decoNote xml:id="b1">Two leather covers.&#xD; Two pairs of sewing
-                           stations.</decoNote>
+                     <binding contemporary="true">
+                        <decoNote xml:id="b1">Two leather boards. Four sewing stations.</decoNote>
                         <decoNote xml:id="b2" type="bindingMaterial">
                            <material key="leather"/>
                         </decoNote>
-                        <decoNote xml:id="b3" type="Endbands"/>
-                        <decoNote xml:id="b4" type="Other">Small holes are visible in the centre
-                           fold of the quires, one close to the head and one close to the tail of
-                           the codex.</decoNote>
-                        <decoNote xml:id="b5">None.</decoNote>
                      </binding>
                   </bindingDesc>
                </physDesc>
                <history>
                   <origin>
-                     <origPlace>
-                        <placeName ref="INS0039MY"/>
-                     </origPlace>
-                     <origDate notBefore="1730" notAfter="1760">mid-18th cent. (?)&#xD; According to
-                        the colophon of QSM-001, <persName role="scribe" ref="PRS12076ZawaldaMaryam">Zawalda Māryām</persName> produced the Ms. in 1745.</origDate>
+                      <origDate notBefore="1730" notAfter="1760">Ca. 1730-60.</origDate>
```

Comment:
`</origDate>` must only be closed once, below, take the closing tag out in this line, or the file is invalid